modelon/impact/client/experiment_definition.py

```diff
@@ -4,10 +4,30 @@
 from modelon.impact.client import exceptions
 
 
-def _assert_valid_args(fmu, custom_function, solver_options, simulation_options):
-    if not isinstance(fmu, entities.ModelExecutable):
-        raise TypeError("Fmu must be an instance of ModelExecutable class!")
-    if not isinstance(custom_function, entities.CustomFunction):
+def _parse_options(default_options, options):
```

Comment:
The name of this function is a bit missleading as it does not parse anything. I think just `_get_options` is fine.